### PR TITLE
Add instructions and script on how to use the lumbar rootlets model

### PR DESCRIPTION
**Pull request by valosekj**
This PR adds instructions and script on how to use the lumbar rootlets model (alpha version).

### CODE REVIEW

File: packaging_lumbar_rootlets/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+## Getting started
+
+⚠️ This README provides instructions on how to use the model for **_lumbar_** rootlets. 
+Please note that this model is still under development and is not yet available in the Spinal Cord Toolbox (SCT).
+
+⚠️ For the stable model for dorsal cervical rootlets only, use SCT v6.2 or higher (please refer to this [README](..%2FREADME.md)).
```
Comment (jcohenadad) [inline packaging_lumbar_rootlets/README.md:6]:
```suggestion
> [!NOTE]
> If you would like to use the model for _dorsal_ cervical rootlets only, use SCT v6.2 or higher (please refer to this [README](..%2FREADME.md)).
```

File: packaging_lumbar_rootlets/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+### Dependencies
+
+- [Spinal Cord Toolbox (SCT)](https://spinalcordtoolbox.com/user_section/installation.html)
+- [conda](https://conda.io/projects/conda/en/latest/user-guide/install/index.html) 
+- Python
```
Comment (jcohenadad) [inline packaging_lumbar_rootlets/README.md:12]: version requirement needed

File: packaging_lumbar_rootlets/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+3. Install the required packages with the following command:
+```
+cd model-spinal-rootlets
+pip install -r packaging_lumbar_rootlets/requirements.txt
+```
+ 
+### Step 3: Getting the Predictions
+
+ℹ️ To temporarily suppress warnings raised by the nnUNet, you can run the following three commands in the same terminal session as the above command:
```
Comment (jcohenadad) [inline packaging_lumbar_rootlets/README.md:46]:
```suggestion
> [!NOTE]
> To temporarily suppress warnings raised by the nnUNet, you can run the following three commands in the same terminal session as the above command:
> ```bash
> export nnUNet_raw="${HOME}/nnUNet_raw"
> export nnUNet_preprocessed="${HOME}/nnUNet_preprocessed"
> export nnUNet_results="${HOME}/nnUNet_results"
> ```
```
I'm not sure this formatting is going to work...

File: packaging_lumbar_rootlets/README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+For example:
+
+```bash
+python packaging_lumbar_rootlets/run_inference_single_subject.py -i sub-001_T2w.nii.gz -o sub-001_T2w_label-rootlets_dseg.nii.gz -path-model ~/Downloads/Dataset202_LumbarRootlets_r20240527 -fold 0
+```
+
+ℹ️ The script also supports getting segmentations on a GPU. To do so, simply add the flag `--use-gpu` at the end of the above commands. 
+By default, the inference is run on the CPU. It is useful to note that obtaining the predictions from the GPU is significantly faster than the CPU.
```
Comment (jcohenadad) [inline packaging_lumbar_rootlets/README.md:69]:
```suggestion
> [!NOTE]
> The script also supports getting segmentations on a GPU. To do so, simply add the flag `--use-gpu` at the end of the above commands. By default, the inference is run on the CPU. It is useful to note that obtaining the predictions from the GPU is significantly faster than the CPU.
```

File: packaging_lumbar_rootlets/README.md (hunk 3 above)
Comment (valosekj) [reply packaging_lumbar_rootlets/README.md:46]: Yes, it is working! https://github.com/ivadomed/model-spinal-rootlets/blob/main/packaging_lumbar_rootlets/README.md#step-3-getting-the-predictions

**Comment by valosekj** [comment]
@jcohenadad, thanks for the suggestions. I apologize for merging the PR too quickly. So, I have addressed your comments in [this commit](https://github.com/ivadomed/model-spinal-rootlets/commit/7b3cd8b9275e1a18da975a7cbbabea08552de9b8).

**Pull request merged**